### DISCUSSION TITLE
Does it works with Wii U pro controller Afterglow?

### DISCUSSION THREAD

**Discussion by Crixtl** [KeyPuncher/WiinUPro#178] [Help]
a friend give me his old wii u, loong story short i did not use it for too long, but i kinda want o adapt the controller for pc, i followed all the steps, but the program cant recognice the device, so i investigate the controller and found out that is a 3rd party controller, i wonder if it could be still positble to troubleshoot.

**Comment by KeyPuncher** [comment]
It does not support the Afterglow controller or many (or all?) third party controllers. They seem to have some sort masking or don't actually implement standard Bluetooth (I actually don't know the technical details behind it) for their Bluetooth which makes them undiscoverable by Windows.
